src/lib/collab/provider.js

```diff
@@ -285,7 +286,13 @@ export class CollabProvider {
     this.updateParticipant(sessionId, data.timestamp)
     this.emit('telepointer', data)
   }
-
+  /**
+   * We receive an update of the schema from the server
+   * We reload the page
+   */
+  onReceiveSchemaUpdated = () => {
+    window.location.reload(true)
```

Comment:
Should we check that all the modifications have been saved ? Can the user lose data if the schema update happen while she is modifying the document ?

Reply:
I tried to explain that in the commit message 

"There is a potential risk of losing the latest editions from Alice (if
Alice network is slow the realtime event can be slow to arrive) but we
accept it since this case should not happen very often."

Reply:
Oups sorry, I checked all the changes at once this time 😅  Thanks for the explanation.

Reply:
I learned from someone that writing a good commit message is essential, I follow this advice! I can now have more lines in a commit message than in the code itself. 

To be more precise about the shortcut we took, we have a discussion about a potential solution to reduce at the minimum the risk of loosing data but it requires more work (and need to be more challenged), something like: 
- When we detect from the opening event that the schema change, before updating the schema, send a message to the channel saying "make a save, I'll refresh you in a second". Then all subscribers should send their latest steps, waits for reconciliation if needed, and then send back "i'm ready". When we receive all these messages, then we can apply the migration and the refresh. 

We decided that for now, the way I handle it in this PR, is enough.